lib/src/code/code.dart

```diff
@@ -75,29 +79,19 @@ class Code {
           isReadOnly: isReadOnly,
         ),
       );
+
       charIndex += line.length + 1;
+      lineIndex++;
     }
 
     return result;
   }
 
   static List<String> _getCommentWords(
```

Comment:
if this function is so simple, is it needed?

Reply:
We may want to strip commas or something like that later. This is also a different level of abstraction. A function should not mix different levels of abstractions.